### ISSUE TITLE
Cannot read property 'bref' of undefined

### ISSUE THREAD

**Issue by bkuhl** [brefphp/bref#954]
**Description:**

I've been tinkering with bref for a bit this evening and have operated successful deployments.  After dealing with a few distractions, I came back and started getting this error when running `serverless deploy`.

```
  TypeError: Cannot read property 'bref' of undefined
      at ServerlessPlugin.addCustomIamRoleForVendorArchiveDownload (/Users/ben/Personal/patchbot.io/vendor/bref/bref/index.js:98:45)
      at PluginManager.invoke (/Users/ben/.nvm/versions/node/v12.13.1/lib/node_modules/serverless/lib/classes/PluginManager.js:576:20)
      at async PluginManager.spawn (/Users/ben/.nvm/versions/node/v12.13.1/lib/node_modules/serverless/lib/classes/PluginManager.js:598:5)
      at async Object.before:deploy:deploy [as hook] (/Users/ben/.nvm/versions/node/v12.13.1/lib/node_modules/serverless/lib/plugins/deploy.js:49:11)
      at async PluginManager.invoke (/Users/ben/.nvm/versions/node/v12.13.1/lib/node_modules/serverless/lib/classes/PluginManager.js:576:9)
      at async PluginManager.run (/Users/ben/.nvm/versions/node/v12.13.1/lib/node_modules/serverless/lib/classes/PluginManager.js:634:7)
      at async Serverless.run (/Users/ben/.nvm/versions/node/v12.13.1/lib/node_modules/serverless/lib/Serverless.js:431:5)
      at async /Users/ben/.nvm/versions/node/v12.13.1/lib/node_modules/serverless/scripts/serverless.js:685:9

     For debugging logs, run again after setting the "SLS_DEBUG=*" environment variable.
```

I tried setting the debug env var but it didn't provide any additional information into the exception.  I'm sure I'm doing something stupid, but I'm unsure what.

```
$ serverless -v
Framework Core: 2.46.0
Plugin: 5.3.0
SDK: 4.2.3
Components: 3.12.0
```

I'm using `brefphp` v1.2.6

**How to reproduce:**

Below is my config.  I only recall making changes to the env vars, so I'd appreciate any insight into what could be the issue here.

```
plugins:
  - ./vendor/bref/bref

service: laravel

provider:
  name: aws
  region: us-east-1
  lambdaHashingVersion: 20201221
  # The stage of the application, e.g. dev, production, staging… ('dev' is the default)
  stage: dev
  runtime: provided.al2
  environment:
    # Some env vars removed
    SQS_QUEUE: !Ref SendMessagesQueue
    # If you create the queue manually, the `SQS_QUEUE` variable can be defined like this:
    # SQS_QUEUE: https://sqs.us-east-1.amazonaws.com/your-account-id/my-queue
  iam:
    role:
      statements:
        # Allows our code to interact with SQS
        -   Effect: Allow
            Action: [sqs:SendMessage, sqs:DeleteMessage]
            Resource: !GetAtt SendMessagesQueue.Arn

package:
    # Directories to exclude from deployment
    exclude:
        - node_modules/**
        - public/storage
        - resources/assets/**
        - storage/**
        - tests/**

functions:
  worker:
    reservedConcurrency: 10
    image: 364782219106.dkr.ecr.us-east-1.amazonaws.com/laravel-test:lambda-serverless
    events:
      # Declares that our worker is triggered by jobs in SQS
      -   sqs:
            arn: !GetAtt SendMessagesQueue.Arn
            # If you create the queue manually, the line above could be:
            # arn: 'arn:aws:sqs:us-east-1:1234567890:my_sqs_queue'
            # Only 1 item at a time to simplify error handling
            batchSize: 1

resources:
  Resources:

    # The SQS queue
    SendMessagesQueue:
      Type: AWS::SQS::Queue
      Properties:
        ContentBasedDeduplication: true
        FifoQueue: True
        QueueName: ${self:service}-SendMessages.fifo
        RedrivePolicy:
          maxReceiveCount: 1 # jobs will be retried up to 1 time
          # Failed jobs (after the retries) will be moved to the other queue for storage
          deadLetterTargetArn: !GetAtt DeadLetterQueue.Arn

    # Failed jobs will go into that SQS queue to be stored, until a developer looks at these errors
    DeadLetterQueue:
      Type: AWS::SQS::Queue
      Properties:
        ContentBasedDeduplication: true
        FifoQueue: True
        QueueName: ${self:service}-Failed.fifo
        MessageRetentionPeriod: 1209600 # maximum retention: 14 days
```

**Comment by t-richard** [comment]
Hi, this is a regression in serverless framework and has been fixed in bref 1.2.7

Could you please update bref and see if it fixes the issue ?

See : https://github.com/brefphp/bref/commit/71f8821088532d46ad61afde17fa1260a61ec450

**Comment by bkuhl** [comment]
That did it, thanks so much!
